### PR TITLE
GroupCreateCommandParser and GroupDeleteCommandParser class

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

**Review by froststein — APPROVED** [review]
Looks good to merge

File: src/main/java/seedu/splitlah/parser/commandparser/GroupCreateCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import seedu.splitlah.exceptions.InvalidFormatException;
+import seedu.splitlah.parser.Parser;
+import seedu.splitlah.parser.ParserUtils;
+import seedu.splitlah.ui.Message;
+
+
+/**
+ * Represents a command parser that is able to parse user arguments into a GroupCreateCommand object.
+ *
+ * @author Roy
```
Comment (ivanaitzliddat) [inline src/main/java/seedu/splitlah/parser/commandparser/GroupCreateCommandParser.java:14]: Perhaps might want to change the author name here to yourself

**Review by ivanaitzliddat — CHANGES_REQUESTED** [review]
Looks good overall, just another very very minor change

**Review by ivanaitzliddat — APPROVED** [review]
Looks good to merge

**Pull request merged**